src/ProjectTemplates/BlazorWasm.ProjectTemplates/content/BlazorWasm-CSharp/Shared/Models/ClaimValue.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace BlazorWasm_CSharp.Shared.Models
```

Comment:
How would you feel about renaming this namespace (and the containing directory) to `Authorization`?

It would be great to avoid a generic term like `Models` if possible as it doesn't really mean anything here. Pretty much everything in the "shared" project will be some sort of "model class".

Reply:
That's fine with me.